test/integration/spec/localtracks.js

```diff
@@ -74,4 +105,92 @@ const createLocalTracks = require('../../../lib/createlocaltrack');
       });
     });
   });
+
+  describe('replaceTrack', function() {
```

Comment:
Please make sure to check that, after re-acquiring a Track:
* `started` is fired
* `enabled` is preserved
* Re-acquire logic also works for re-acquired Tracks.